lightblue-ldap-config/src/main/java/com/redhat/lightblue/config/ldap/LdapDBResolver.java

```diff
@@ -68,4 +71,20 @@ private LdapDataSourceConfiguration findByDatabase(String database){
         return null;
     }
 
+	@Override
+	public List<LDAPConnection> getConnections() {
+		
+		List<LDAPConnection> ldapConnections = new ArrayList<>();
+		
+		for (LdapDataSourceConfiguration ldapDS : ldapDataSources){
+			try {
+				ldapConnections.add(ldapDS.getLdapConnection());
+			} catch (LDAPException e) {
+				ldapConnections.add(null);
```

Comment:
Can we do something other than store nulls in the list when the connection fails? Maybe just for the LDAPException in the list, if the health check response can handle it this would allow us to add error details which could help debug/recover faster.